apps/web/.env.example

```
@@ -27,18 +27,42 @@ AUTH_DEV_LOGIN=true
+# and a region are all present, and the sign-in page renders the interim form
+# alone whenever it returns nothing. COGNITO_REGION falls back to AWS_REGION.
+# COGNITO_CLIENT_SECRET is needed only for an app client configured with one.
+#
+# Setting these does NOT retire dev login on its own — AUTH_DEV_LOGIN above is
+# a separate switch, and it stays on until the whole cohort can sign in.
+#
+# University SSO, when it lands, is a Cognito change and not an application
+# change: the IdP becomes an upstream provider on this same user pool and
+# Cognito stays the token issuer. No new variable appears here for it.
+COGNITO_USER_POOL_ID=
+COGNITO_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COGNITO_CLIENT_ID key should go before the COGNITO_USER_POOL_ID key

(UnorderedKey)